### DISCUSSION TITLE
more customizable hud appearing options?

### DISCUSSION THREAD

**Discussion by Axilotl17** [ppy/osu#17978] [Ideas] [answered]
this might be a duplicate (im not very good at checking for those, although i do try) but, I would like to have a few things visible during gameplay (example: just accuracy and combo) but i dont want to see other things (like, progress bar). but, i would prefer to see everything at the pauses. is this coming with the new skinning features, maybe?

**Comment by peppy** [comment, marked as answer]
This is already planned to be a per-element setting.
